core/store/models/job_spec.go

```diff
@@ -263,6 +263,33 @@ type InitiatorParams struct {
 	Requesters AddressCollection `json:"requesters,omitempty" gorm:"type:text"`
 	Name       string            `json:"name,omitempty"`
 	Params     string            `json:"-"`
+	FromBlock  *Big              `json:"fromBlock,omitempty" gorm:"type:varchar(255)"`
+	ToBlock    *Big              `json:"toBlock,omitempty" gorm:"type:varchar(255)"`
+	Topics     Topics            `json:"topics,omitempty" gorm:"type:text"`
+}
+
+type Topics [][]common.Hash
+
+func (t Topics) Scan(value interface{}) error {
+	jsonStr, ok := value.(string)
+	if !ok {
+		return fmt.Errorf("Unable to convert %v of %T to Topics", value, value)
+	}
+
+	err := json.Unmarshal([]byte(jsonStr), &t)
+	if err != nil {
+		return fmt.Errorf("Unable to convert %v of %T to Topics: %v", value, value, err)
```

Comment:
This is good, but maybe errors.Wrap(..) would be better?
https://godoc.org/github.com/pkg/errors#Wrapf